jenkins-pipelines/performance_staging/perf-regression-scylla-gradual-throughput-grow-gauss.jenkinsfile

```diff
@@ -6,9 +6,9 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 perfRegressionParallelPipeline(
     backend: "aws",
     aws_region: "us-east-1",
-    test_name: "performance_regression_gradual_grow_throughput.PerformanceRegressionGradualGrowThroughutTest",
-    test_config: '''["test-cases/performance/perf-regression-gradual-throughput-grow.yaml", "configurations/cs-gauss-distribution.yaml"]''',
-    sub_tests: ["test_write_gradual_increase_load","test_read_gradual_increase_load", "test_mixed_gradual_increase_load"],
+    test_name: "performance_regression_gradual_grow_throughput.PerformanceRegressionGradualGrowThroughputTest",
+    test_config: '''["test-cases/performance/perf-regression-gradual-throughput-grow.yaml", "configurations/cs-gauss-distribution.yaml", "configurations/performance/cassandra_stress_gradual_load_steps_enterprise.yaml", "configurations/disable_kms.yaml", "configurations/tablets_disabled.yaml", "configurations/disable_speculative_retry.yaml"]''',
```

Comment:
You should remove the "configurations/cs-gauss-distribution.yaml" from the config.
I don't think we use it anywhere (or should use).